src/Featurizers/UnitTests/ShortGrainDropperFeaturizer_UnitTests.cpp

```diff
@@ -0,0 +1,190 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include "../../Featurizers/ShortGrainDropperFeaturizer.h"
+#include "../TestHelpers.h"
+#include "../../Archive.h"
+
+namespace NS = Microsoft::Featurizer;
+
+void TestImpl(std::vector<std::vector<std::vector<std::string>>> trainingBatches,
```

Comment:
>TestImpl [](start = 5, length = 8)

I'd expect to see more tests that exercise the estimator's parameters. For example, consider writing a series of tests that illustrate how results change with the same training set data when different parameters are used. The parameters should be selected to exercise a variety of different scenarios. Test SECTIONs will make the implementation a bit easier ([https://github.com/catchorg/Catch2/blob/master/docs/tutorial.md#test-cases-and-sections](https://github.com/catchorg/Catch2/blob/master/docs/tutorial.md#test-cases-and-sections))